packages/headless/src/database/mongodb/store-branch/tempStoreBranch.ts

```diff
@@ -0,0 +1,47 @@
+import { model, Schema } from 'mongoose';
+import { randomUUID } from 'crypto';
+import { StoreBranch } from 'src/entity/store-branch';
+import { BranchAddressSchema } from '../branch/branch.model';
+
+const TmpStoreBranchSchema = new Schema<StoreBranch>(
+  {
+    id: {
+      type: String,
+      default: () => randomUUID(),
+      unique: true,
+    },
+    // storeId
+    store: {
+      type: String,
+      index: true,
+    },
+    url: {
+      type: String,
+      unique: true,
+    },
+    info: {
+      shopName: {
+        type: String,
+        unique: true,
```

Comment:
don't make it unique here. because, if the owner won't be able to creates two branch create request